src/Groups/directproducts.jl

```diff
@@ -167,6 +210,46 @@ end
     embedding(G::DirectProductGroup, j::Int)
 
 Return the embedding of the `j`-th component of `G` into `G`, for `j` = 1,...,#factors of `G`.
+
+# Examples
+```jldoctest
+julia> H = symmetric_group(3)
+Sym( [ 1 .. 3 ] )
+
+julia> K = symmetric_group(2)
+Sym( [ 1 .. 2 ] )
+
+julia> G = direct_product(H,K)
+DirectProduct of
+ 
```

Comment:
This output is bad. It hints at a bug in the `Base.show` method for direct products. Indeed, it looks like this:
```julia
function Base.show(io::IO, G::DirectProductGroup)
   if G.isfull
      print(io, "DirectProduct of ")
      display(G.L)
   else
      print(io, String(GAP.Globals.StringViewObj(G.X)))
   end
end
```
So it invokes `display`, which ignores the `io` object. One can reproduce the problem in the REPL using sprint:
```
julia> str = sprint(show, G);
2-element Vector{PermGroup}:
 Sym( [ 1 .. 3 ] )
 Sym( [ 1 .. 2 ] )

julia> str
"DirectProduct of "

```

I think this can be fixed by calling `show(io, "text/plain", G.L)` instead of `display(G.L)`. Or perhaps a better printing could be used: the output
```
DirectProduct of 2-element Vector{PermGroup}:
 Sym( [ 1 .. 3 ] )
 Sym( [ 1 .. 2 ] )
```
is not really great. How about
```
DirectProduct of
 Sym( [ 1 .. 3 ] )
 Sym( [ 1 .. 2 ] )
```
achieved by writing something akin to `for x in G.L  print(io, "\n  ",x) end`.

Could you add such a fix and then update the test outputs to match?